docs/whats-new/dotnet-docs-mod0.md

```diff
@@ -1,93 +1,129 @@
+- [What's new in .NET 8](../core/whats-new/dotnet-8.md) - Updated for RC 1
+- [Tutorial: Debug a .NET console application using Visual Studio Code](../core/tutorials/debugging-with-visual-studio-code.md) - .NET 8 update for VS Code tutorial series
+- [Tutorial: Create a .NET class library using Visual Studio Code](../core/tutorials/library-with-visual-studio-code.md) - .NET 8 update for VS Code tutorial series
+- [Tutorial: Publish a .NET console application using Visual Studio Code](../core/tutorials/publishing-with-visual-studio-code.md) - .NET 8 update for VS Code tutorial series
+- [Tutorial: Test a .NET class library using Visual Studio Code](../core/tutorials/testing-library-with-visual-studio-code.md) - .NET 8 update for VS Code tutorial series
+- [Tutorial: Create a .NET console application using Visual Studio Code](../core/tutorials/with-visual-studio-code.md) - .NET 8 update for VS Code tutorial series
 
 ## .NET tools and diagnostics
 
 ### New articles
 
-- [.NET observability with OpenTelemetry](../core/diagnostics/observability-with-otel.md)
-- [SYSLIB0053: AesGcm should indicate the required tag size](../fundamentals/syslib-diagnostics/syslib0053.md)
-- [Specialized diagnostics](../core/diagnostics/specialized-diagnostics-overview.md)
-- [.NET diagnostic tools](../core/diagnostics/tools-overview.md)
-- [NETSDK1195: Trimming, or code compatibility analysis for trimming, single-file deployment, or ahead-of-time compilation is not supported for the target framework.](../core/tools/sdk-errors/netsdk1195.md)
+- [NETSDK1083: The specified RuntimeIdentifier is not recognized](../core/tools/sdk-errors/netsdk1083.md)
+- [CA1870: Use a cached 'SearchValues' instance](../fundamentals/code-analysis/quality-rules/ca1870.md)
+- [SYSLIB0052: APIs that support obsolete mechanisms for Regex extensibility are obsolete](../fundamentals/syslib-diagnostics/syslib0052.md)
 
-### Updated articles
+## Advanced .NET programming
+
+### New articles
 
-- [Collect metrics](../core/diagnostics/metrics-collection.md) - Edit pass of collect metrics
+- [Source generation for ComWrappers](../standard/native-interop/comwrappers-source-generation.md)
+- [Implicit method signature translations in .NET interop](../standard/native-interop/preserve-sig.md)
 
 ## C# language
 
 ### New articles
 
-- [Errors and warnings associated with source generators and interceptors](../csharp/language-reference/compiler-messages/source-generator-errors.md)
-- [C# standard specification](../csharp/language-reference/specifications.md)
-- [Tutorial: Explore primary constructors](../csharp/whats-new/tutorials/primary-constructors.md)
+- [Compiler Error CS8964](../csharp/language-reference/compiler-messages/cs8964.md)
+- [Resolve errors and warnings generated from expressions prohibited in expression trees](../csharp/language-reference/compiler-messages/expression-tree-restrictions.md)
```

Comment:
Add the updated article on sync vs. async (stored) exceptions. @gewarren I found it.

```suggestion
- [Resolve errors and warnings generated from expressions prohibited in expression trees](../csharp/language-reference/compiler-messages/expression-tree-restrictions.md)

### Updated articles

- [Add section on exceptions in task-returning methods](../csharp/[fundamentals/exceptions/creating-and-throwing-exceptions.md)

```